docker-compose.yml

```diff
@@ -124,6 +136,11 @@ services:
       SECURE_COOKIE: "false"
       SHOW_SAML_ERRORS: "true"
       THEME_USE: "material:material"
+      SESSION_STORE_TYPE: "sql"
+      MYSQL_HOST: "db"
+      MYSQL_DATABASE: "silauth"
+      MYSQL_USER: "silauth"
+      MYSQL_PASSWORD: "silauth"
```

Comment:
Should this match the password on line 25?

Reply:
Line 25 is `MYSQL_ROOT_PASSWORD`. I assume it's for the root user rather than for the `silauth` user.

I took this from simplesamlphp-module-silauth/docker-compose.yml.